dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -36,19 +37,32 @@
 
   private static final Logger logger = Logger.getLogger(DynatraceMetricExporter.class.getName());
 
-  private DynatraceMetricExporter(URL url, String apiToken, Boolean enrichWithOneAgentMetaData) {
+  private DynatraceMetricExporter(
+      URL url,
+      String apiToken,
+      String prefix,
+      Labels defaultDimensions,
+      Boolean enrichWithOneAgentMetaData) {
     this.url = url;
     this.apiToken = apiToken;
 
-    Collection<AbstractMap.SimpleEntry<String, String>> localTags = new ArrayList<>();
+    Collection<AbstractMap.SimpleEntry<String, String>> localDimensions = new ArrayList<>();
 
     if (enrichWithOneAgentMetaData) {
       OneAgentMetadataEnricher enricher = new OneAgentMetadataEnricher(logger);
-      localTags.addAll(enricher.getDimensionsFromOneAgentMetadata());
+      localDimensions.addAll(enricher.getDimensionsFromOneAgentMetadata());
     }
 
+    if (defaultDimensions != null) {
+      defaultDimensions.forEach(
+          (String k, String v) -> {
+            localDimensions.add(new AbstractMap.SimpleEntry<>(k, v));
```

Comment:
Are these deduplicated later?

Reply:
yes, in `MetricAdapter.getInstance().setTags(...);`